Makefile

```diff
@@ -1,5 +1,8 @@
 .PHONY: test docs
 
+COVERAGE_OPTS := --rcfile $(CURDIR)/.coveragerc
```

Comment:
We also need it for the `coverage merge` command to work (which is what `pytest-cov` does internally).